### ISSUE TITLE
Performance: store object types for static objects in voxels.

### ISSUE THREAD

**Issue by mklingen** [CompletelyFairGames/dwarfcorp#745]
We currently only query the octree for objects directly. Most of the time this is very wasteful. If we just stored object types in voxels and only queried when needed, we could save a lot of our performance overhead that planning  uses. Most of the time knowing the exact identity of an object (as opposed to its type) isn't even necessary.

**Comment by com1clyf3** [comment]
Regressing. This get solved or ongoing?

**Comment by com1clyf3** [comment]
@mklingen  Close?

**Comment by mklingen** [comment]
No, It should still be done sometime, I just don't care enough to make it work right now.
